.travis.yml

```diff
@@ -17,6 +17,6 @@ cache:
     - packages/workflow-angular/node_modules
     - packages/workorder-angular/node_modules
     - packages/sync/node_modules
-    - examples/step-vehicle-inspection/node_modules
-    - examples/step-accident/node_modules
+    - steps/examples/step-vehicle-inspection/node_modules
+    - steps/examples/step-accident/node_modules
```

Comment:
step-signature module does not need to be added here?

Reply:
Thanks for reminder! Adding!